src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -74,6 +87,39 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
     });
   }
 
+  get shouldDisplayCellReference(): boolean {
+    if (this.isCellReferenceVisible) {
+      return true;
+    }
+    const { sheetId, col, row } = this.env.model.getters.getActivePosition();
+    const zone = this.env.model.getters.expandZone(sheetId, positionToZone({ col, row }));
+    const rect = this.env.model.getters.getVisibleRect(zone);
+    if (
+      !deepEquals(rect, this.rect) ||
+      sheetId !== this.env.model.getters.getCurrentEditedCell()?.sheetId
+    ) {
+      this.isCellReferenceVisible = true;
+    }
+    return this.isCellReferenceVisible;
+  }
+
+  get cellReference(): string {
+    const { col, row, sheetId } = this.env.model.getters.getCurrentEditedCell();
+    const prefixSheet = sheetId !== this.env.model.getters.getActiveSheetId();
+    return `${prefixSheet ? this.env.model.getters.getSheetName(sheetId) + "!" : ""}${toXC(
+      col,
+      row
+    )}`;
+  }
+
+  get cellReferenceStyle(): string {
+    const { x: left, y: top } = this.rect;
+    return `
+      left: ${left - 1}px;
```

Comment:
could you cast this as contants ? why -1 ?

Reply:
cast this as contants ? What do you mean ?
The -1 is here because it's prettier